code/messaging/src/phone/java/com/adobe/marketing/mobile/MessagingUtils.java

```diff
@@ -187,6 +160,22 @@ private static Variant getVariantValue(final Object value) {
         return convertedValue;
     }
 
+    static List<PropositionPayload> createPropositionPayload(final List<Map<String, Object>> payloads) {
```

Comment:
since this method is returning a list of `PropositionPayload` objects, it's probably better named `getPropositionPayloads`